mlir/unittests/Dialect/QCO/IR/test_unitary_op_interface.cpp

```
@@ -0,0 +1,120 @@
+    qco::QCOProgramBuilder builder(context.get());
+    builder.initialize();
+    buildFunc(builder);
+    return builder.finalize();
+  }
+
+  /**
+   * @brief Get first operation of given type in a module containing a function
+   *        as its first operation.
+   */
+  template <typename OpType>
+  [[nodiscard]] OpType getFirstOp(ModuleOp moduleOp) {
```

Cppcheck (2.19.0)

[style] The function 'getFirstOp' is never used.

(unusedFunction)